bedrock/firefox/templates/firefox/browsers/compare/ie.html

```diff
@@ -89,7 +89,7 @@ <h2>{{ ftl('compare-shared-security-and-privacy-strong') }}</h2>
 
       <img src="{{ static('img/firefox/compare/compare-privacy-locks.svg') }}" alt="" height="217" width="320" class="compare-details-image">
 
-      <p>{{ ftl('compare-ie-so-whats-the-solution-if-your', fallback='compare-ie-so-whats-the-solution-if-your-fallback', lockwise='href="%s" data-cta-type="link" data-cta-name="Lockwise"'|safe|format(url('firefox.products.lockwise')),
+      <p>{{ ftl('compare-ie-so-whats-the-solution-if-your', fallback='compare-ie-so-whats-the-solution-if-your-fallback', lockwise='href="%s" data-cta-type="link" data-cta-name="Lockwise"'|safe|format(url('firefox.features.password-manager')),
```

Comment:
You changed the cta-name in the other two but not this one

Reply:
good catch. thanks.